src/main.ts

```
@@ -446,9 +445,16 @@ export default class ChatGptViewProvider implements vscode.WebviewViewProvider {
           }
           break;
         case BackendMessageType.getSettings:
-          const getSettingsData = data as GetSettingsMessage;
           this.frontendMessenger.sendSettingsUpdate(vscode.workspace.getConfiguration("chatgpt"));
           break;
+        case BackendMessageType.getViewOptions:
+          const viewOptions = await this.offlineStore?.getViewOptions();
+          this.frontendMessenger.sendViewOptionsUpdate(viewOptions ?? {});
+          break;
+        case BackendMessageType.setViewOptions:
+          const viewOptionsData = data as SetViewOptionsMessage;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)